packages/rum/src/userActionCollection.ts

```diff
@@ -21,7 +21,8 @@ export interface CustomUserAction {
   type: UserActionType.CUSTOM
   name: string
   startTime: number
-  context?: Context
+  context: Context
+  globalContext: Context
```

Comment:
it seems weird to me that only this event type contains the global context.
Why not propagating the global context like on the logs side `[CustomUserAction, Context]`?

by having:
```ts
 addUserActionStrategy(action, context)
```

we could also only copy contexts before init and use current ones after init. 
Or is there a reason to also copy them after init?

Reply:
So, if we use `[CustomUserAction, Context]`, this means that we would have a `LifeCycle.subscribe` signature like:

```ts
  subscribe(eventType: LifeCycleEventType.CUSTOM_ACTION_COLLECTED, callback: (data: [CustomUserAction, Context]) => void): Subscription
```

Does it looks right to you? I'd prefer to keep having an object instead of a tuple.

Reply:
I don't really mind but we could use `data: {action: CustomUserAction, context: Context}` if you prefer an object